addons/web/static/src/js/core/registry.js

```diff
@@ -69,7 +69,8 @@ odoo.define("web.Registry", function (require) {
          * @returns {Object}
          */
         entries() {
-            return Object.assign({}, this.map);
+            const proto = Object.getPrototypeOf(this.map) || {};
```

Comment:
this looks wrong... it only works one level, but you could have a registry inheriting from another which inherit from yet another